### PR TITLE
Fix/11498 request tan call button

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Adding missing accessibility label

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11489

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift [hunk 1]
```diff
@@ -105,7 +105,7 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 							style: .headline,
 							color: .enaColor(for: .textTint),
 							title: AppStrings.ExposureSubmissionHotline.phoneNumberDomestic,
-							accessibilityLabel: AppStrings.ExposureSubmissionHotline.callButtonTitle,
+							accessibilityLabel: "\(AppStrings.ExposureSubmissionHotline.callButtonTitle):\n\n\(AppStrings.ExposureSubmissionHotline.phoneNumberDomestic)",
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:108]: please move to `AccessibilityIdentifiers`

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift [hunk 2]
```diff
@@ -122,7 +122,7 @@ class ExposureSubmissionHotlineViewController: DynamicTableViewController, ENANa
 							style: .headline,
 							color: .enaColor(for: .textTint),
 							title: AppStrings.ExposureSubmissionHotline.phoneNumberForeign,
-							accessibilityLabel: AppStrings.ExposureSubmissionHotline.callButtonTitle,
+							accessibilityLabel: "\(AppStrings.ExposureSubmissionHotline.callButtonTitle):\n\n\(AppStrings.ExposureSubmissionHotline.phoneNumberForeign)",
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:125]: same here

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift (hunk 1 above)
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:108]: Why are the new lines needed?

Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift:108]: I wonder about formatting here. Won't it make more sense to adde a new AppString?
And as @ArturFriesen  mentioned I also wonder about the newlines.

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionHotlineViewController.swift]: @KaiTeuber I removed the line breaks, I would suggest to use the current strings and not create a new string, reason: if we need to change the number, we have to do at two different places and if we forget to change then the voice over will be different to that of the exact number.

**Review by KaiTeuber — APPROVED** [review]

**Pull request merged**